bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java

```diff
@@ -35,7 +35,7 @@
  * </p>
  */
 
-@Version("5.4.1") // aligning this with release/650 since af2-rest-api is compiled with 5.2.0 in release/650
+@Version("5.5.0") // aligning this with release/650 since af2-rest-api is compiled with 5.2.0 in release/650
```

Comment:
why are we bumping the version?

Reply:
Since I added a new API that's why I increased package version. Is this not required ?